app/services/dual-output/dual-output.ts

```diff
@@ -626,6 +619,8 @@ export class DualOutputService extends PersistentStatefulService<IDualOutputServ
           corruptedNodeIds.add(corruptedNode.id);
```

Comment:
i think i missed this before but since `validateSource` always returns the `partnerNode` doesn't that mean every node will have its partner marked as a corrupted node?

Reply:
validate source only repairs and returns the node if the scene items do not have the same source